classes/02class/exercises/c02-network06/afaryy/README.md

```diff
@@ -0,0 +1,91 @@
+# c02-network06
+
+
+- Create an Elastic IP:
+```bash
+export AWS_PROFILE=devopsacademyau
+aws sts get-caller-identity
```

Comment:
Why do you need this? Were you just checking if using the correct user?

Reply:
It is just a habbit to check again.